test_runner/src/main/kotlin/ftl/args/ValidateIosArgs.kt

```diff
@@ -99,6 +101,11 @@ private fun IosArgs.validType() {
         throw FlankConfigurationError("Type should be one of ${validIosTypes.joinToString(",")}")
 }
 
+private fun IosArgs.assertXcTestRunVersion() {
+    if (!onlyTestConfiguration.isNullOrEmpty() or !skipTestConfiguration.isNullOrEmpty())
+        if (xcTestRunData.version == XcTestRunVersion.V1) throw FlankConfigurationError("Specified [xctestrun-file] doesn't contain test plans. Options: [only-test-configuration] or [skip-test-configuration] are not valid for this [xctestrun-file]")
+}
```

Comment:
I think that you could write it in single if

Reply:
IMO complex conditions are always not convenient for reading. What is the difference between 

```
private fun IosArgs.assertXcTestRunVersion() {
    if (!onlyTestConfiguration.isNullOrEmpty() or !skipTestConfiguration.isNullOrEmpty())
        if (xcTestRunData.version == XcTestRunVersion.V1) throw FlankConfigurationError(
            "Specified [xctestrun-file] doesn't contain test plans. Options: [only-test-configuration] or [skip-test-configuration] are not valid for this [xctestrun-file]"
        )
}
```

and

```
private fun IosArgs.assertXcTestRunVersion2() {
    if ((!onlyTestConfiguration.isNullOrEmpty() or !skipTestConfiguration.isNullOrEmpty())
        and xcTestRunData.version == XcTestRunVersion.V1) throw FlankConfigurationError(
            "Specified [xctestrun-file] doesn't contain test plans. Options: [only-test-configuration] or [skip-test-configuration] are not valid for this [xctestrun-file]"
        )
}

It almost the same ;P
```

Reply:
so my proposition is to move complex condition into a separate function, IMO nested ifs are not so readable